modules/opcua_server_module/src/opcua_server_impl.cpp

```diff
@@ -11,7 +11,7 @@ using namespace daq;
 using namespace daq::opcua;
 
 OpcUaServerImpl::OpcUaServerImpl(DevicePtr rootDevice, PropertyObjectPtr config, const ContextPtr& context)
-    : Server("OpenDAQOPCUAServerModule", config, rootDevice, context, nullptr)
+    : Server("OpenDAQOPCUA", config, rootDevice, context, nullptr)
```

Comment:
So it seems that "ServerImpl" is the only user-addable component that accepts the config as a constructor parameter. That feels a bit odd.

Reply:
before discovery implementation  each child of server was storing the config inside.